data/stoa0244d/stoa005/stoa0244d.stoa005.opp-lat1.xml

```diff
@@ -40,24 +42,28 @@
       </publicationStmt>
       
       <sourceDesc>
-        <listBibl xml:lang="lat">
+        <listBibl>
           <biblStruct>
             <monogr>
               <editor>
                 <persName>
-                  <name xml:lang="lat">Augustus Engelbrecht</name>
-                  <addName xml:lang="de">August Engelbrecht</addName>
+                  <name>August Engelbrecht</name>
                 </persName>
               </editor> 
-              <author ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0244d">Rufinus</author>  
-              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0244d.stoa005">Interpretatio Orationum Gregorii Nazianzeni</title>
+              <author ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0244d">Rufinus of Aquileia</author>  
+              <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0244d.stoa005">Tyranni Rufini Orationum Gregorii Nazianzeni Novem Interpretatio</title>
               <imprint>
-                <publisher>Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
-                <pubPlace>Wien, Österreich</pubPlace>
+                <publisher>Tempsky</publisher>
+                <publisher>Freytag</publisher>
+                <pubPlace>Vienna</pubPlace>
```

Comment:
My question  would be the same as above, because I think for some of those early Loebs we have publisher, pubPlace, publisher, pubPlace? I know it's all over the place, but that grouping seems to be more intuitive to me.

Reply:
Because I love to be inconsistently consistent I believe that I have done it both ways, though in the CSEL corpus I'm pretty sure I have mostly done the two pubPlace names and then the two publisher names.  As I said above happy to go back through for consistency now that all the files are finally done.

Reply:
Yes, in looking at canonical Greek I think we present it differently (and in varied ways there, too!). This way might be confusing if there was some automated scraping of this data since the association is unclear. I would just flag it and add the best practice to the CTS metadata info guidelines.
https://github.com/PerseusDL/tei-conversion-tools/wiki/cts-textgroup-and-work-metadata-files
I realize this is undocumented and it's a good case for clarification.

Reply:
That is an excellent plan, for now I will merge this pull request and we can also add this to my list of things to programatically change.